onyo/lib/git.py

```diff
@@ -109,7 +109,7 @@ def files(self) -> list[Path]:
         return self._files
 
     def clear_cache(self) -> None:
-        """Clear cache of this instance of GitRepo.
+        r"""Clear cache of this instance of GitRepo.
```

Comment:
"GitRepo" should probably be in backticks?

Reply:
Thanks for the catch. I have fixed it locally, but will not include it in this PR.